tests/unit/Storage/ABS/SnowflakeExportAdapterTest.php

```diff
@@ -71,23 +92,36 @@ public function testGetCopyCommandCompressed(): void
     TIMESTAMP_FORMAT = 'YYYY-MM-DD HH24:MI:SS'
 )
 MAX_FILE_SIZE=50000000
+DETAILED_OUTPUT = TRUE
 EOT
             ,
             []
-        );
+        )->willReturn($expectedCopyResult);
 
         $source = new Storage\Snowflake\Table('schema', 'table');
         $options = new ExportOptions(true);
         $adapter = new Storage\ABS\SnowflakeExportAdapter($conn);
-        $adapter->runCopyCommand(
-            $source,
-            $destination,
-            $options
+
+        $this->assertSame(
+            $expectedCopyResult,
+            $adapter->runCopyCommand(
+                $source,
+                $destination,
+                $options
+            )
         );
     }
 
     public function testGetCopyCommandQuery(): void
     {
+        $expectedCopyResult = [
+            [
+                'FILE_NAME' => 'containerUrl',
```

Comment:
Teď jsem zrovna narazil na to, že je na prd mít "dummy" stringy v mock responses. Měl by tam být skutečný string. Snadno se na tom pak ukáže pokud vevnitř něco s tím stringem něco dělá a dělá to blbě (třeba z toho stripuje http nebo koncové lomítko a tak).